packages/client/hmi-client/src/components/models/Model.vue

```diff
@@ -283,24 +307,41 @@ import Splitter from 'primevue/splitter';
 import SplitterPanel from 'primevue/splitterpanel';
 import Toolbar from 'primevue/toolbar';
 import TeraResizablePanel from '../widgets/tera-resizable-panel.vue';
-import { example } from './example-model-extraction'; // TODO - to be removed after March demo
 
-export interface ModelProps {
-	assetId: string;
-	isEditable: boolean;
-	highlight?: string;
-}
+const emit = defineEmits(['create-new-model', 'update-tab-name']);
 
-const extractions = ref(Object.values(example));
+const extractions = ref([]);
 
-const props = defineProps<ModelProps>();
+const props = defineProps({
+	assetId: {
+		type: String,
+		default: '',
+		required: true
+	},
+	isEditable: {
+		type: Boolean,
+		default: false,
+		required: true
+	},
+	highlight: {
+		type: Boolean,
+		default: false,
+		required: false
+	}
+});
```

Comment:
I think generally if you have required=true, you don't want to specify defaults.